go/tools/builders/env.go

```diff
@@ -60,6 +60,8 @@ type env struct {
 	workDirPath string
 
 	shouldPreserveWorkDir bool
+
+	execDir string
```

Comment:
It doesn't look like this is actually used anymore. Can it be removed?

Reply:
Good catch, removed